pages/common/git-reset.md

```diff
@@ -0,0 +1,11 @@
+# git reset
+
+> Undo commits. Reset current HEAD to the specified state.
+
+- Undo last commit, keep the changes:
```

Comment:
Just to be clearer about where the changes are kept, I'd be explicit and say "keep the changes in the filesystem".